app/context/OrbisContext.tsx

```diff
@@ -77,15 +77,16 @@ export const ODB = ({ children }: OrbisDBProps) => {
           Buffer.from(storedSession, "base64").toString(),
         ) as { cacao: Cacao };
 
+        const issuedAt = Date.parse(cacao.p.iat);
         const expTime = cacao.p.exp;
         const sessionAddress = cacao.p.iss
           .replace("did:pkh:eip155:1:", "")
           .toLowerCase();
 
         if (
           sessionAddress !== walletClient.account.address.toLowerCase() ||
-         ( expTime !== undefined &&
-          Date.parse(expTime) < Date.now()
+          (expTime !== undefined && Date.parse(expTime) < Date.now()) ||
+          issuedAt < Date.now() - 2 * 60 * 60 * 1000
```

Comment:
Let's use a constant for the 2hrs.

Reply:
Good idea! Done in 4071d35f973531bdf1e3af67edac0d1f83d9af7e